src/trio/_threads.py

```diff
@@ -252,9 +254,9 @@ def run_in_system_nursery(self, token: TrioToken) -> None:
 
 
 @enable_ki_protection  # Decorator used on function with Coroutine[Any, Any, RetT]
```

Comment:
I assume this comment was just wrong and can be removed? (It reads like explaining the `# type: ignore[misc]`)

Reply:
Yeah, wasn't 100% clear at first, but I think you're right